resources/js/contest-voting/entry.coffee

```diff
@@ -61,12 +61,11 @@ export class Entry extends React.Component
       if @props.contest.show_votes
         div className: 'contest-voting-list__title contest-voting-list__title--show-votes',
           div className: 'contest-voting-list__votes-bar', style: { width: "#{relativeVotePercentage}%" }
-          div className: 'u-relative u-ellipsis-overflow', entry_title
-          @renderUserLink()
+          @renderTitle(entry_title)
       else
         div className: 'contest-voting-list__title',
-          div className: 'u-ellipsis-overflow', entry_title
-          @renderUserLink()
+        @renderTitle(entry_title)
```

Comment:
can be inlined

Reply:
Removed `entry_title` since the `a` tag is used only in the title anyways